docs/project/os-onboarding.md

```diff
@@ -14,8 +14,9 @@ Continuing with the idea of pragmatism, if you only read this far, you've got th
 
 References:

```

Comment:
It feels like the above section is leading with the "why" rather than the "what" -- that is, it is explaining our reasoning before mentioning the strategy. Should we invert that for people who are only interested in the "what"?